### PR TITLE
Reflect the introduction of RISCOF

### PR DESCRIPTION
**Pull request by 2011eric**
Since this project has migrated to RISCOF,
the information in README is also updated.

### CODE REVIEW

**Review by jserv — CHANGES_REQUESTED** [review]
Read https://cbea.ms/git-commit/ and write better git commit messages.

File: README.md [hunk 1]
```diff
@@ -48,56 +53,6 @@ make quake
 
 The usage and limitations of Doom and Quake demo are listed in [docs/demo.md](docs/demo.md).
 
-## riscv-arch-test
```
Comment (jserv) [inline README.md:51]: The section should be kept along with revisions for new RISCOF framework.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ access to all of its features. Your target system must have a functional [SDL2 l
 * macOS: `brew install sdl2`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev`
 
+Install [RISCOF](https://riscof.readthedocs.io/en/stable/installation.html#install-riscof).
+```shell
+pip3 install riscof
```
Comment (jserv) [inline README.md]: Check https://github.com/sysprog21/rv32emu/blob/master/.ci/riscv-tests.sh for RISCOF installation. We shall instal `riscof` from git repository for the sake of latest development.

File: README.md (hunk 1 above)
Comment (2011eric) [reply README.md:51]: I see.

File: README.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ access to all of its features. Your target system must have a functional [SDL2 l
 * macOS: `brew install sdl2`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev`
 
+Install [RISCOF](https://riscof.readthedocs.io/en/stable/installation.html#install-riscof).
```
Comment (Risheng1128) [inline README.md:26]: I think the url should use the home page of riscof. Like the following:
```
[RISCOF](https://riscof.readthedocs.io/en/stable/)
```

Comment (2011eric) [reply README.md:26]: Got it.

Comment (jserv) [reply README.md:26]: `riscof` is only required if you want to execute riscv-arch-tests. So, move to the dedicated section.

Comment (2011eric) [reply README.md:26]: https://github.com/sysprog21/rv32emu/blob/b70dcaa86dac3adc76b24a939fb7414906e2447e/README.md?plain=1#L71-L74

What about the old description of running risv-arch-test?

Comment (jserv) [reply README.md:26]: Exactly. Improve this description.

Comment (2011eric) [reply README.md:26]: Is there a way to run the old framework now?

Comment (jserv) [reply README.md:26]: No, old 2.x framework is deprecated. We shall no longer rely on it.

**Comment by jserv** [comment]
https://github.com/RinHizakura/riscv-emulator/commits/main # Recently, @RinHizakura is moving toward RISCOF for his RISC-V emulator. @2011eric, check if the wording in `README.md` is precise for rv32emu.

**Comment by Risheng1128** [comment]
In [neorv32-riscof](https://github.com/stnolting/neorv32-riscof), there is some description of RISCOF in `README.md`.

File: README.md [hunk 4]
```diff
@@ -95,10 +97,12 @@ Current progress of this emulator in riscv-arch-test(RV32):
     - `F` Standard Extension for Single-Precision Floating-Point
 
 Detail in riscv-arch-test:
+* [RISCOF document](https://riscof.readthedocs.io/en/stable/)
 * [riscv-arch-test repository](https://github.com/riscv-non-isa/riscv-arch-test)
 * [RISC-V Architectural Testing Framework](https://github.com/riscv-non-isa/riscv-arch-test/blob/master/doc/README.adoc)
 * [RISC-V Architecture Test Format Specification](https://github.com/riscv-non-isa/riscv-arch-test/blob/master/spec/TestFormatSpec.adoc)
 
+
```
Comment (jserv) [inline README.md]: Don't change this line.

**Pull request merged**